plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/properties/KuduTableProperties.java

```diff
@@ -69,7 +66,7 @@
 
     private static final ObjectMapper mapper = new ObjectMapper();
 
-    private static final long DEFAULT_DEADLINE = 20000; // deadline for retrieving range partitions in milliseconds
+    private static final long DEFAULT_TIMEOUT = 20_000; // timeout for retrieving range partitions in milliseconds
```

Comment:
nit : Can we capture them as a separate commit ?

Reply:
This change is because the method KuduTable.getRangePartitions(long timeout) use argument name timeout.

Reply:
Thanks. How about renaming the commit message - `Use KuduTable#getRangePartitions for partition details`